PC/VS9.0/pyproject.vsprops

```diff
@@ -12,7 +12,7 @@
 		InlineFunctionExpansion="1"
 		EnableIntrinsicFunctions="true"
 		AdditionalIncludeDirectories="..\..\Include; ..\..\PC"
-		PreprocessorDefinitions="_WIN32"
+		PreprocessorDefinitions="_WIN32;WIN32"
```

Comment:
Actually this is unrelated to just Expat.  It brings the VS9.0 project files inline with the MSBuild ones.  Ever wonder why the Expat change "just worked" on the other releases? ;)